### DISCUSSION TITLE
Generating config files for sweets running in radar coordinates

### DISCUSSION THREAD

**Discussion by Pinoquio1891** [opera-adt/COMPASS#228] [Q&A]
Hi!
Currently I am working with point targets (retro reflectors) and in order to avoid any interpolation artifacts the aim is to run all analysis in radar coordinates. As far as I understood, in principle, Dolphin would be adequate for monitoring / analyzing positions in this case. It appears Dolphin accepts co-registered stacks as input I could create e.g. with ISCE2 or maybe Sweets / Compass? In order to maintain things within one framework I would prefer using either Sweets  or Compass. Looking at the code I saw that compass can generate co-registered images in radar coordinates passing -g radar to s1_cslc but I did not see any possibility to already add this flag in the config file generation e.g. s1_geocode_stack, which sweets calls - and in deed it might seem strange to be able to generate radar coordinate output with a function / workflow named 'geocode', but in order to not to duplicate code could be the most obvious way. Or is there any other possibility yet to generate these stacks with compass? Maybe @hfattahi can give some hint?
Cheers
Christian

**Comment by scottstanie** [comment]
Hi, just checking what you're looking to do- you want to run sweets, meaning go all the way up the processing chain to unwrapped interferograms, but do it only for one burst in radar coordinates? Or were you also hoping to have multiple bursts stitched together geographically at some stage, but still do the early parts in radar coordinates?

**Reply by Pinoquio1891** [reply]
Hi, what I want to do at the end is to run Dolphin. The idea about Sweets was just to run the first two steps in order to get a co-registered stack of SLCs I can feed into Dolphin. Currently I am using ISCE2 with MiaPlPy / Mintpy where ISCE2 creates the stack and as my current site / ROI is at a burst border ISCE2 unpacks two bursts and does burst overlap and secondaries to reference co-registration. My final signal (time series) evaluation is still in radar coordinates, therefore I just skip the final geocoding step of Mintpy.
